lib/httpserver/httpserver.go

```diff
@@ -30,9 +30,10 @@ import (
 )
 
 var (
-	tlsEnable   = flag.Bool("tls", false, "Whether to enable TLS (aka HTTPS) for incoming requests. -tlsCertFile and -tlsKeyFile must be set if -tls is set")
-	tlsCertFile = flag.String("tlsCertFile", "", "Path to file with TLS certificate. Used only if -tls is set. Prefer ECDSA certs instead of RSA certs as RSA certs are slower. The provided certificate file is automatically re-read every second, so it can be dynamically updated")
-	tlsKeyFile  = flag.String("tlsKeyFile", "", "Path to file with TLS key. Used only if -tls is set. The provided key file is automatically re-read every second, so it can be dynamically updated")
+	tlsEnable       = flag.Bool("tls", false, "Whether to enable TLS (aka HTTPS) for incoming requests. -tlsCertFile and -tlsKeyFile must be set if -tls is set")
+	tlsCertFile     = flag.String("tlsCertFile", "", "Path to file with TLS certificate. Used only if -tls is set. Prefer ECDSA certs instead of RSA certs as RSA certs are slower. The provided certificate file is automatically re-read every second, so it can be dynamically updated")
+	tlsKeyFile      = flag.String("tlsKeyFile", "", "Path to file with TLS key. Used only if -tls is set. The provided key file is automatically re-read every second, so it can be dynamically updated")
+	tlsCipherSuites = flag.String("tlsCipherSuite", "", "Defines a list of cipher suites for example. It can be defined as tls cipher names (TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA) or uint16 numbers (0x2f,0x35). Used only if -tls is set. The provided ciphers suites is automatically re-read every second, so it can be dynamically updated")
```

Comment:
```suggestion
	tlsCipherSuites = flag.String("tlsCipherSuite", "", "Defines a list of comma-separated cipher suites. It can be defined as tls cipher names (TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA) or uint16 numbers (0x2f,0x35). Used only if -tls is set. The provided ciphers suites are automatically re-read every second, so it can be dynamically updated")
```

Btw, what it means "The provided ciphers suites are automatically re-read every second, so it can be dynamically updated" ? Are these suits placed on the instance somewhere?

Reply:
updated description